diff --git a/dist/chunks/cac.BuBILSID.js b/dist/chunks/cac.BuBILSID.js
index be0c6aa27e88c3bce62d20d0a1c55d7d24a93a03..468944a710c9b290fc2cba78306e3e8e9f183a88 100644
--- a/dist/chunks/cac.BuBILSID.js
+++ b/dist/chunks/cac.BuBILSID.js
@@ -690,7 +690,6 @@ const cliOptionsConfig = {
 	},
 	silent: {
 		description: "Silent console output from tests. Use `'passed-only'` to see logs from failing tests only.",
-		argument: "[value]",
 		transform(value) {
 			if (value === "true" || value === "yes" || value === true) return true;
 			if (value === "false" || value === "no" || value === false) return false;
